plans/tasks.py

```diff
@@ -24,11 +26,21 @@ def autorenew_account(providers=None):
     PLANS_AUTORENEW_BEFORE_HOURS = getattr(settings, "PLANS_AUTORENEW_BEFORE_HOURS", 0)
 
     accounts_for_renewal = get_active_plans().filter(
-        userplan__recurring__has_automatic_renewal=True,
-        userplan__recurring__token_verified=True,
-        userplan__expire__lt=datetime.date.today()
-        + datetime.timedelta(
-            days=PLANS_AUTORENEW_BEFORE_DAYS, hours=PLANS_AUTORENEW_BEFORE_HOURS
+        (
```

Comment:
Two subsequent `filter()` calls instead of `&` might make this a bit clearer.

Reply:
Replaced with double-`filter`